packages/ecs-agent/0005-bottlerocket-fix-procfs-path-on-host.patch

```diff
@@ -1,4 +1,4 @@
-From baf491ab24fb2dfc1c4a98c8492fc32769475cc6 Mon Sep 17 00:00:00 2001
+From d8ce3933e19713ff5ef02d9ed4b22e59150f9103 Mon Sep 17 00:00:00 2001
```

Comment:
This is a patch that is specific for Bottlerocket. I'm not sure on the feasibility of getting the ECS agent to have special logic for Bottlerocket hosts upstream so I think this might be a patch we carry indefinitely.

Reply:
Makes sense!